scripts/lib/install_opencv.sh

```diff
@@ -1,13 +1,23 @@
 #
-# Install opencv and its dependencies
+# Install opencv and related dependencies
 
 install_opencv() {
   # Update
   sudo apt-get -y update
-  
-  # Install pip
+ 
+  # Installing dependencies 
   sudo apt install python3-pip
+
+  sudo apt install python3-venv
+  
+  cd ../src/cv/
+
+  # Setting up new virtual environment
+  python3 -m venv venv
+  
+  source venv/bin/activate
+
+  # Install opencv requirements
+  pip install -r opencv_requirements.txt
```

Comment:
Change this filename to requirements.txt

Reply:
changed